client/src/components/login/components/registration/registration.js

```diff
@@ -2,33 +2,62 @@ import React, { Component } from 'react'
 import './registration.css'
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
 import { faUser } from '@fortawesome/free-solid-svg-icons';
+import axios from 'axios';
+import { API_URL } from '../loginPanel/config'
```

Comment:
can we change this api url variable name?

Reply:
what would be your proposal ?

Reply:
authentication_url?